examples/query/react/mutations/src/features/posts/PostDetail.tsx

```diff
@@ -29,7 +29,10 @@ const EditablePostName = ({
   onCancel: () => void
   isLoading?: boolean
 }) => {
-  const [name, setName] = useState(initialName)

```

Comment:
even with adding the `useEffect`, this line is still required I think?

Reply:
@phryneas , yes, that line is required, that's my bad.